Doc/library/exceptions.rst

```diff
@@ -562,9 +562,12 @@ The following exceptions are the exceptions that are usually raised.
 
    Raised when the interpreter finds an internal error, but the situation does not
    look so serious to cause it to abandon all hope. The associated value is a
-   string indicating what went wrong (in low-level terms).
+   string indicating what went wrong (in low-level terms). In :term:`CPython`,
+   this could be raised by incorrectly using Python's C API, such as returning
+   a ``NULL`` value without an exception set.
 
-   You should report this to the author or maintainer of your Python interpreter.
+   If you're confident that this exception wasn't your fault, you should
+   report this to the author or maintainer of your Python interpreter.
```

Comment:
You should also mention package authors. The only way this could be the user's fault is if they use ctypes or write C/C++/Rust code themselves. But they may be using a package.